### ISSUE TITLE
Update checker still checking SourceForge

### ISSUE THREAD

**Issue by jimmiedave** [openrocket/openrocket#825]
Noted that on startup, OpenRocket is still checking SourceForge for updates. The update URL is apparently specified in net.sf.openrocket/communication (not a Java coder; please forgive if this path to the trouble isn't well-specified).

I'm assuming it should be checking github.

Also noting that "sourceforge" appears a few times in the code elsewhere, and many times in "web" documentation that's in the repo.

```
public abstract class Communicator {

	protected static final String BUG_REPORT_URL;

	protected static final String UPDATE_INFO_URL;
	
	static {
		String url;
		url = System.getProperty("openrocket.debug.bugurl");
		if (url == null)
			url = "http://openrocket.sourceforge.net/actions/reportbug";
		BUG_REPORT_URL = url;
		
		url = System.getProperty("openrocket.debug.updateurl");
		if (url == null)
			url = "http://openrocket.sourceforge.net/actions/updates";
		UPDATE_INFO_URL = url;
	}

```

**Comment by wolsen** [comment]
I think this all needs a bit of rework. The update url returns a 404 and I'm not sure of the infrastructure that's serving this. Will need to look at it closer.

**Comment by neilweinstock** [comment]
Suggestion: remove it completely until it works properly with updated mechanism.

**Comment by neilbalch** [comment]
I'm interested in potentially working on this one. Is there an analogous set of resources we'd like it to point people to on GitHub?

I presume that the bug tracker should link to the GitHub issues tracker and the updater should check the GitHub releases page...

**Comment by jimmiedave** [comment]
This is where GitHub needs a "like" button. Awesome!

DC

On January 28, 2022 1:22:32 PM EST, Joe Pfeiffer ***@***.***> wrote:
>Closed #825 via #1082.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/openrocket/openrocket/issues/825#event-5972596306
>You are receiving this because you authored the thread.
>
>Message ID:
***@***.***>
